doc/classes/Input.xml

```diff
@@ -312,6 +312,106 @@
 				[b]Note:[/b] Due to keyboard ghosting, [method is_physical_key_pressed] may return [code]false[/code] even if one of the action's keys is pressed. See [url=$DOCS_URL/tutorials/inputs/input_examples.html#keyboard-events]Input examples[/url] in the documentation for more information.
 			</description>
 		</method>
+		<method name="joy_adaptive_triggers_feedback">
+			<return type="void" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<param index="2" name="position" type="int" />
+			<param index="3" name="strength" type="int" />
+			<description>
+				Starts "Feedback" adaptive triggers effect on the specified joypad. This method will fail if the joypad is not DualSense.
+				Axis should be either [constant JOY_AXIS_TRIGGER_LEFT] for the left trigger or [constant JOY_AXIS_TRIGGER_RIGHT] for the right trigger.
+				[code skip-lint]position[/code] parameter can take values from 0 to 9 (inclusive).
+				[code skip-lint]strength[/code] parameter can take values from 0 to 8 (inclusive).
+				[b]Note:[/b] This feature is only supported on Windows, Linux and macOS.
+			</description>
+		</method>
+		<method name="joy_adaptive_triggers_multi_feedback">
+			<return type="void" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<param index="2" name="strengths" type="int[]" />
+			<description>
+				Starts "Multi Feedback" adaptive triggers effect on the specified joypad. This method will fail if the joypad is not DualSense.
+				Axis should be either [constant JOY_AXIS_TRIGGER_LEFT] for the left trigger or [constant JOY_AXIS_TRIGGER_RIGHT] for the right trigger.
+				[code skip-lint]strengths[/code] array can have at most 10 elements, each of them can take values from 0 to 8 (inclusive).
```

Comment:
```suggestion
				[code skip-lint]strengths[/code] array must have exactly 10 elements, and each of them can take values from 0 to 8 (inclusive).
```
The current wording implies the array could have fewer than 10 elements; however, passing an array of not exactly 10 elements causes an error.